contents/tutorials/stripe-reports.md

```diff
@@ -0,0 +1,185 @@
+---
+title: How to set up Stripe reports
+date: 2024-06-21
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+For your startup to become a real business, it needs to make money. Stripe is the most popular choice for helping you do this. Creating and analyzing reports for your Stripe data helps you understand how you are making money and how you can improve.
+
+This tutorial shows you how to sync your Stripe data to PostHog and then create a report of insights like customer count, gross revenue, recurring revenue, revenue churn, and more.
+
+## Linking Stripe data to PostHog
+
+To start, you need both a Stripe and PostHog account. Once have those, head to PostHog's [data warehouse tab](https://us.posthog.com/data-warehouse) and click **Link source**. Choose the Stripe option by clicking **Link**, and then enter your account ID and client secret, and press **Next**. Keep all tables selected and click **Import**. 
```

Comment:
It would be better if they create a restricted key in stripe specifically for posthog, since the client secret grants us all sorts of access. I suspect we only need read access to a few things. Also, we should link to where they should create it: https://dashboard.stripe.com/apikeys/create